.github/workflows/main.yml

```
@@ -120,3 +120,31 @@ jobs:
           build-args: |
             APP_SHA=${{ github.sha }}
             VERSION=main
```

Would it be a good time to change it?
```suggestion
            VERSION=${{ github.sha }}
```